bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -73,17 +71,14 @@
  * @author Miguel Álvarez - Register audio source and refactor
  */
 @NonNullByDefault
-public class PulseaudioHandler extends BaseThingHandler implements DeviceStatusListener {
+public class PulseaudioHandler extends BaseThingHandler {
 
     public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
             .unmodifiableSet(Stream.of(SINK_THING_TYPE, COMBINED_SINK_THING_TYPE, SINK_INPUT_THING_TYPE,
                     SOURCE_THING_TYPE, SOURCE_OUTPUT_THING_TYPE).collect(Collectors.toSet()));
     private final Logger logger = LoggerFactory.getLogger(PulseaudioHandler.class);
-    private final int refresh = 60; // refresh every minute as default
 
-    private @Nullable PulseaudioBridgeHandler bridgeHandler;
-    private @Nullable String name;
-    private @Nullable ScheduledFuture<?> refreshJob;
+    private @NonNullByDefault({}) String name;
```

Comment:
If I do that, I must confess it has side effect.  The "null" value is actually a marker in the method PulseaudioBridgeHandler.getChildHandlers() : If the name is null, then it means that the handler is not initialized, and the method filter it out. 
Now that you said it, I agree it's bad practice (AND it's a bug because name is not set to null when disposed).
Do you know something to check if a handler is initialized, or should I use something else (a boolean, or the childHandledInitialized/disposed event system ?)

Reply:
I decided to use the childHandledInitialized methods in my last commit, and so I put the name to an empty string to avoid null checking as you said.